apps/site/assets/css/_trip-plan-results.scss

```diff
@@ -24,7 +24,10 @@
     background-color: $brand-primary-lightest-contrast;
     border: 1px solid $brand-primary;
     border-bottom: 0;
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
I wonder if it makes sense to set these flexbox properties only for wider viewports, so we don't get wrapping in the mobile view